sklearn/preprocessing/_encoders.py

```diff
@@ -155,7 +155,11 @@ class OneHotEncoder(_BaseEncoder):
     strings, denoting the values taken on by categorical (discrete) features.
     The features are encoded using a one-hot (aka 'one-of-K' or 'dummy')
     encoding scheme. This creates a binary column for each category and
-    returns a sparse matrix or dense array.
+    returns a sparse matrix
+    (A matrix with most of if the elements equal to zero)
```

Comment:
I am -1 for this change. We could link to the glossary instead: https://scikit-learn.org/stable/glossary.html#term-sparse-matrix.

```
returns a :term:`sparse matrix` or dense array depending on the value of
the `sparse` parameter.

Reply:
i could change my PR and add the glossary link @glemaitre but i suggest instead of giving the link we can give the description wouldn't it be more handy?